processing-tests/common/test/kotlin/com/livefront/sealedenum/compilation/location/OutsideSealedClassTests.kt

```diff
@@ -82,8 +97,9 @@ class OutsideSealedClassTests {
     fun `outside enum ordering`() {
         assertEquals(
             listOf(
-                GammaOutsideSealedClassEnum.GammaFirstObject,
                 GammaOutsideSealedClassEnum.GammaOutsideSealedClass_GammaSecondObject,
+                GammaOutsideSealedClassEnum.GammaFirstObject,
+                GammaOutsideSealedClassEnum.GammaFourthObject,
                 GammaOutsideSealedClassEnum.GammaThirdObject,
```

Comment:
This is the primary breaking change, since the order is now the inner object first (`GammaSecondObject`) and then the other objects (in alphabetical order).